packages/nimble-components/src/menu/tests/menu.stories.ts

```diff
@@ -262,19 +246,24 @@ export const nestedMenu: StoryObj<MenuArgs> = {
 };
 
 export const customMenu: StoryObj<MenuArgs> = {
-    parameters: {
-        docs: {
-            description: {
-                story: 'The menu can be configured to display a custom header with arbitrary content.  This could include other nimble components, or even custom elements.'
-            }
-        }
-    },
     // prettier-ignore
     render: createUserSelectedThemeStory(html`
+        <style>
```

Comment:
This change wasn't necessary as part of this PR, but I noticed that we didn't have our custom menu header content styled using nimble tokens. Since we specifically have docs saying that you can do this, I thought it was important to set a good example by using tokens to style it.